libcudacxx/test/libcudacxx/cuda/cmath.pass.cpp

```diff
@@ -0,0 +1,63 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the libcu++ Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// SPDX-FileCopyrightText: Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES.
+//
+//===----------------------------------------------------------------------===//
+
+#include <cuda/cmath>
+#include <cuda/std/utility>
+#include "test_macros.h"
+
+template <class T>
+void test()
+{
+  constexpr T maxv = cuda::std::numeric_limits<T>::max();
+
+  assert(ceil_div(T(0), T(1)) == T(0));
+  assert(ceil_div(T(1), T(1)) == T(1));
+  assert(ceil_div(maxv, T(1)) == maxv);
+  assert(ceil_div(maxv, maxv) == T(1));
```

Comment:
Suggestion: please add one or two tests for the ordinary use case as well, like `ceil_div(12314, 256)`.

Reply:
I went with a smaller one, because we need to stay in the bounds of `char`

Reply:
Ah yes, makes totally sense!